t.rb

```diff
@@ -0,0 +1,3 @@
+# bad
+[2, 1, 3].sort.first
+[2, 1, 4].sort.first
```

Comment:
Use `min` instead of `sort...first`.

```suggestion
[2, 1, 4].min
```